.github/CONTRIBUTING.md

```diff
@@ -1,151 +1,212 @@
 # Contributing Guidelines
 
-## <a name="question"></a> Got a Question?
+## Got a Question?
 
-Please do not open issues for general support such as "how to do reaction roles" — use the
-[support server](https://discord.gg/4udtcA5) for that.
+Please do not open issues for general support such as "how to do reaction roles"—read the docs and/or ask on the
+[support server] for that.
+
+[support server]: https://discord.gg/4udtcA5
 
 To keep this tracker clean, we will systematically close issues regarding general support and redirect to the
 aforementioned support server.
 
-## <a name="bug"></a> Found a Bug?
+## Found a Bug?
 
 Do **not** use this tracker to file maybe-bugs in YAGPDB. We have a dedicated channel on the support server for that, to
 which you can post after it was confirmed to be one.
 
 Should there be general issues with the documentation (grammatical structure, general flow, clarification needed), do
 feel free to open an issue clearly outlining what is wrong about it and consider suggesting how to fix it.
 
-## Submission Guidelines
+## Submission Guide
+
+Thanks for contributing to the YAGPDB documentation! Please take a moment to review this document in order to make the
+contribution process easy and effective for everyone involved.
 
 ### Requirements
 
-- An [editorconfig](https://editorconfig.org)-capable (text) editor.
-- The latest extended release of [Hugo](https://gohugo.io).
-- Optional: a webserver to serve a release build of the website.
+You will at the very least need an [EditorConfig]-capable (text) editor like VSCode or Neovim. Though the GitHub web
+interface may work fine, you cannot save in-between without having to commit your changes, which is not ideal.
```

Comment:
```suggestion
Small changes can be authored directly using the GitHub web interface, though we highly recommend an [EditorConfig]-capable editor like VS Code or Neovim for any extended work.
```
"need an editor" and "the GitHub web interface may work fine" are slightly contradictory :-)